source/Irrlicht/CSceneNodeAnimatorCameraMaya.cpp

```diff
@@ -191,6 +194,14 @@ void CSceneNodeAnimatorCameraMaya::animateNode(IDummyTransformationSceneNode *no
 
 	// Rotation ------------------------------------
 
+	auto getValueDependentOnHandOrientation = [&](const float& expression)

```

Comment:
@devshgraphicsprogramming could I create a protected function in ISceneNodeAnimator handling that? DRY

Reply:
nope, that would lead to clutter.

you can acutally use `core::mix(-expression,expression,bool)`